packages/mobile/src/app/reducers.ts

```diff
@@ -196,6 +208,7 @@ export const appReducer = (
         multiTokenUseUpdatedFeed: action.configValues.multiTokenUseUpdatedFeed,
         linkBankAccountEnabled: action.configValues.linkBankAccountEnabled,
         sentryTracesSampleRate: action.configValues.sentryTracesSampleRate,
+        biometryEnabled: action.configValues.biometryEnabled && Platform.OS === 'ios',
```

Comment:
i thought it would be best to add this platform check in the code, so that we can keep using this feature flag in future releases if we want to enable for android

Reply:
I think we can do this from Remote Config as well (have different values for android and iOS) but this is fine as well I suppose :)

Reply:
yeah i think configuring remote config based on OS is the plan! i added this check because in a future release we might enable biometrics for everyone using this feature flag, but for an android user who hasn't upgraded and is stuck on this version of the app still, i don't want biometrics to be enabled for them when the value of this flag changes (we haven't tested Android properly)